fastlane/Fastfile

```diff
@@ -941,7 +957,7 @@ def build_purchases_android_test_apk(package_name, apk_name, build_type, flavor)
       "minSdkVersion" => 21
     }
   )
-  apk_path = "../purchases/build/outputs/apk/androidTest/#{flavor}/#{build_type}/purchases-#{flavor}-#{build_type}-androidTest.apk"
+  apk_path = "../purchases/build/outputs/apk/androidTest/#{flavor}/#{build_type}/purchases-*-androidTest.apk"
```

Comment:
Had to change this because the naming of the APK doesn't match the flavor name now that we have multiple dimensions, this was easier than finding another workaround